src/styles/theme.scss

```diff
@@ -0,0 +1,384 @@
+
+/* ------------------------------
+
+   Variables
+
+------------------------------ */
+
+// Contemplating Action variables
+$purple: #662483;
+$purple-light: #8b1abc;
+$blue-darkest: #1a1e64;
+$blue-dark: #30336d;
+$blue-light: #4575ba;
+$black: #3c3c3b;
+$white: #f5f5f5;
+$grey-light: #eee;
+$grey-light-purple: #e4e1e7;
+$lime-green: #07c103;
+
+// Utility variables
+$white-pure: #ffffff;
+$black-pure: #000000;
+$shadow: rgba(89, 97, 116, 0.4);
+
+// Theme colours
+$primary-colour: $blue-darkest;
+$secondary-colour: $blue-light;
+$text-colour: $black;
+$interaction-colour: $purple;
+$focus-colour: $lime-green;
+
+/* ------------------------------
+
+   Defaults
+
+------------------------------ */
+
+*:focus {
+  outline-color: $focus-colour;
+}
+
+:root {
+  font-size: 100%;
+  font-family: 'Poppins', 'Arial';
```

Comment:
This will need to be the brand font

Reply:
I'm happy for this to happen in #13 - I think it will be more useful to take these out when we're trying to put something else in its place.